src/Aspire.Dashboard/Extensions/TimeProviderExtensions.cs

```diff
@@ -1,24 +1,26 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Dashboard.Model;
+
 namespace Aspire.Dashboard.Extensions;
 
 internal static class TimeProviderExtensions
 {
-    public static DateTime ToLocal(this TimeProvider timeProvider, DateTimeOffset utcDateTimeOffset)
+    public static DateTime ToLocal(this BrowserTimeProvider timeProvider, DateTimeOffset utcDateTimeOffset)
```

Comment:
This narrowing isn't technically required as far as I can tell (`BrowserTimeProvider` is a `TimeProvider`). Perhaps revert the changes in this class, or rename the class to `BrowserTimeProviderExtensions`.

Reply:
I narrowed it because I don't want someone to accidentally use plain of `TimeProvider` by accident. (this helped me catch a few places I had missed)

If there is a scenario where these extension methods are used on a normal provider then we can revisit.

Reply:
Might rename to BrowserTimeProviderExtensions in the future. I won't right now to avoid merge delay.